src/charm.pyi

```diff
@@ -1,4 +1,5 @@
-from typing import Optional, Union, List, Dict, Tuple, TypedDict, NamedTuple
+from typing import Optional, Union, List, Dict, Tuple, TypedDict, NamedTuple, Any
+from dataclasses import dataclass
```

Comment:
I don't see any dataclasses being newly defined? Maybe I missed it

Reply:
Removed, thanks!